ui/app/src/modules/Preview/Tools/AudiencesPanel.tsx

```diff
@@ -225,158 +240,29 @@ export default function AudiencesPanel() {
-
-    onFormChange(name, datetime);
-    timezone && onFormChange(`${name}_tz`, encodeURIComponent(timezone));
-  };
-
-  switch (property.type) {
-    case "dropdown":
-      return (
-        <AudiencesControl property={property}>
-          <Select
-            labelId={property.id}
-            id={property.id}
-            value={modelValue}
-            onChange={handleSelectChange(property.id)}
-            disabled={modelApplying}
-          >
-            {
-              property.values ? (
-                property.values.map((possibleValue: any, index: number) => (
-                  <MenuItem value={possibleValue.value} key={index}>{possibleValue.value}</MenuItem>
-                ))
-              ) : (null)
-            }
-          </Select>
-          <FormHelperText>{property.helpText}</FormHelperText>
-        </AudiencesControl>
-      );
-    case "checkbox-group":
-      const valuesArray = nnou(modelValue) ? modelValue.split(',') : [];
-
-      return (
-        <AudiencesControl property={property}>
-          <>
-            {
-              property.values ? (
-                property.values.map((possibleValue: any, index: number) => (
-                  <FormControlLabel
-                    key={index}
-                    htmlFor={property.id}
-                    control={
-                      <Checkbox
-                        color="primary"
-                        checked={valuesArray.includes(possibleValue.value)}
-                        onChange={handleInputChange(property.id, possibleValue.value, valuesArray)}
-                        disabled={modelApplying}
-                      />
-                    }
-                    label={possibleValue.value}/>
-                ))
-              ) : (null)
-            }
-            <FormHelperText>{property.helpText}</FormHelperText>
-          </>
-        </AudiencesControl>
-      );
-    case "date-time":
-      return (
-        <AudiencesControl property={property}>
-          <>
-            <DateTimePicker
-              initialDate={modelValue}
-              timezone={modelTimezone}
-              onChange={dateTimePickerChange(property.id)}
-              disabled={modelApplying}/>
-            <FormHelperText>{property.helpText}</FormHelperText>
-          </>
-        </AudiencesControl>
-      );
-    default:
-      return (
-        <AudiencesControl property={property}>
-          <TextField
-            id={property.id}
-            type="text"
-            name="input"
-            placeholder="auto"
-            fullWidth
-            value={modelValue}
-            helperText={property.helpText}
-            onChange={handleInputChange(property.id)}
-            disabled={modelApplying}
-          />
-        </AudiencesControl>
-      );
-  }
-}
-
-interface AudiencesControlProps {
-  property: ContentTypeField;
+interface AudiencesFormSectionProps {
+  field: ContentTypeField;
   children: any;
 }
```

Comment:
`type AudiencesFormSectionProps = PropsWithChildren<{ field: ContentTypeField }>;`

(I had shown you this trick a few weeks back, remember? Type it instead of copy-pasting to help with memory.)